### DISCUSSION TITLE
DragDrop does not work after publishing using  NativeAOT .

### DISCUSSION THREAD

**Discussion by sudongg** [AvaloniaUI/Avalonia#6852] [Q&A]
I'm trying to publish apps using NativeAOT .
After release, the software can run normally and mouse events can respond, but DragDrop does not work.

**Comment by maxkatz6** [comment]
As far as I know NativeAOT doesn't work well with COM yet. Or it requires additional configuration.

NativeAOT by itself is experimental technology, which isn't safe to use in production. And Avalonia relies on COM a lot.

**Reply by kekekeks** [reply]
@maxkatz6 we probably need to switch the rest of our COM interop to microcom. It works wonderfully with WinRT APIs.
